### ISSUE TITLE
маркировка заблокированных

### ISSUE THREAD

**Issue by inquisitor6789** [freedomsex/freedomsex#332]
Если анкета заблокирована автоматикой, то при просмотре связанной жалобы она не маркируется как заблокированная (рыжий кружок)
https://github.com/freedomsex/freedomsex/issues/317

**Comment by tebaly** [comment]
Это косяк "отображения", поправим

**Comment by tebaly** [comment]
Нет, не совсем косяк.. Вы верно заметили. На данном этапе сфера блокировок "система/администратор" работают раздельно. Надеюсь в скором времени "объединить".
